### ISSUE TITLE
Ability to display the listed items in grid or list layout

### ISSUE THREAD

**Issue by Ricarditu** [CrocoBlock/suggestions#2384]
Hi!

I've noticed, and confirmed with support, that there is no way of changing the way the listed items are shown from grid to list, and the opposite.

I think this would be  a good feature to implement, an ajax list/grid layout change.

Thank you!

**Comment by raroDev** [comment]
Yes, we have this type of switches in JetWooBuilder, but it is not possible to achieve this with JetEngine when you are compiling with JetSmartFilters. Would be nice to have that.

**Comment by jbriggs923** [comment]
This would be fantastic! My users can view their favorite video posts in a stylish way (big thumbnail & text/terms ect). While this looks great, it starts to become an issue when there is a large list of posts.
